src/components/Modals/PlaceBetModal.tsx

```diff
@@ -131,6 +131,7 @@ const PlaceBetModal: React.FC<Props> = ({
       const tx = await placeBet(selectedMarket, back, wager, signer);
       setTxHash(tx);
     } catch (err: any) {
+      console.error(err);
```

Comment:
If the error is already logged in the web3 error handler won't this cause a third log?

Reply:
This line is here because nothing was getting logged. Maybe it's working now with the refactor.